src/locales/fi.json

```diff
@@ -516,7 +516,17 @@
       "totalSurfaceAreaLong": "Hanke-alueiden kokonaispinta-ala",
       "liikenneverkollinenHaitta": "Hankealueen liikenneverkollinen haitta",
       "haittaMaarittelyt": "Hankealueen omat haittamäärittelyt",
-      "liikennehaittaIndeksit": "Hankealueen liikennehaittaindeksit"
+      "liikennehaittaIndeksit": "Hankealueen liikennehaittaindeksit",
+      "haittojenhallintasuunnitelma": {
```

Comment:
These translations don't seem to be used. Or are they for the future?

Reply:
Hmm. I cannot remember why I added these when the correct texts are already available elsewhere. I removed these. Thanks for noticing!